### ISSUE TITLE
Loopback flow no longer supported by Google

### ISSUE THREAD

**Issue by viktorcode** [hippogamesunity/SimpleGoogleSignIn#14]
Loopback flow sign-in was deprecated by Google. Attempt to use it results in a message saying we no longer can use it. The support should be removed from the code and the documentation updated accordingly, so users wouldn't waste time trying to make it work.

**Comment by hippogamesunity** [comment]
Hello, it was deprecated for mobile clients. The asset uses Loopback flow only for OAuth inside Unity Editor.

**Comment by hippogamesunity** [comment]
Do you mean it doesn't work inside Editor now?

**Comment by viktorcode** [comment]
It doesn't work for the editor, and it doesn't work for Windows desktop, where I encountered the issue.

**Comment by viktorcode** [comment]
According to this https://developers.google.com/identity/protocols/oauth2/resources/loopback-migration all existing clients of loopback flow are now blocked.

**Comment by hippogamesunity** [comment]
March 14, 2022 - new OAuth clients blocked from using the Loopback IP address flow

Ok, you can use default credentials in Editor as the demo app is considered as an 'old' client. I'm note sure if I can do anything else with this.

**Comment by hippogamesunity** [comment]
Can you confirm that your newly created credentials with Desktop type are no longer work, right?

**Comment by hippogamesunity** [comment]
I can also note that it's about mobile clients
![image](https://github.com/hippogamesunity/SimpleGoogleSignIn/assets/7318172/8b2fd79b-5d0a-44e6-9946-b679d76bbf48)

**Comment by hippogamesunity** [comment]
I think you're wrong, here is the proper link
https://developers.google.com/identity/protocols/oauth2/native-app#redirect-uri_loopback

**Comment by hippogamesunity** [comment]
![image](https://github.com/hippogamesunity/SimpleGoogleSignIn/assets/7318172/32d246bd-c996-4321-8643-68ab5a8db96e)

**Comment by hippogamesunity** [comment]
Can I delete the issue?

**Comment by viktorcode** [comment]
I can confirm that my credentials, which are new, don't work on Windows desktop app (macOS of course works as a mobile). I get the error from Google on authorisation attempt explicitly stating that loopback flow no longer supported. And there was a link to migration guide in this error. 

The link I posted may be different because I searched for it later. But effect is the same: it can no longer work for Windows, so I'm sticking to another way of authorisation there. 

All I wanted is to let you know about that. Thank you, the issue can be closed / deleted.
